packages/sfeed/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE='https://codemadness.org/sfeed-simple-feed-parser.html'
+TERMUX_PKG_DESCRIPTION='Shell-script/crontab oriented feed aggregator and parser utility (curses-based reader included)'
+TERMUX_PKG_LICENSE='ISC'
+TERMUX_PKG_MAINTAINER='@termux'
+## ref: https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=sfeed
+TERMUX_PKG_VERSION='1.6'
+TERMUX_PKG_SRCURL='https://codemadness.org/releases/sfeed/sfeed-'$TERMUX_PKG_VERSION'.tar.gz'
+TERMUX_PKG_SHA256=bfd6d24ce98619726aa411a7a0d969806ad08a73c9adc3a8d04508e00eea6aea
+TERMUX_PKG_DEPENDS='ncurses'
+TERMUX_PKG_RECOMMENDS='curl'
+TERMUX_PKG_SUGGESTS='lynx, termux-tools'
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_MAKE_INSTALL_TARGET="PREFIX=$TERMUX_PREFIX MANPREFIX=$TERMUX_PREFIX/share/man install"
```

Comment:
TERMUX_PKG_EXTRA_MAKE_ARGS would be more suitable for these variables (since they aren't a make install target). Looks like the Makefile in any case needs a patch to make it use system $CPPFLAGS, so you could patch it so that it respects PREFIX at the same time, i.e. having `PREFIX ?= /usr/local`, and thereby get rid of this line

Reply:
Thanks for review and comments!  Modified the commit according your comments.  It is may fault that send a PR w/o testing on Github CI first (the local build-package.sh does success tho).  Now it should works.